AsyncDisplayKit/Layout/ASLayoutableValidation.mm

```diff
@@ -0,0 +1,105 @@
+//
+//  ASLayoutableValidation.mm
+//  AsyncDisplayKit
+//
+//  Copyright (c) 2014-present, Facebook, Inc.  All rights reserved.
+//  This source code is licensed under the BSD-style license found in the
+//  LICENSE file in the root directory of this source tree. An additional grant
+//  of patent rights can be found in the PATENTS file in the same directory.
+//
+
+#import "ASLayoutableValidation.h"
+#import "ASLayout.h"
+#import "ASDisplayNode.h"
+
+#import "ASLayoutableValidationBlockProvider.h"
+#import "ASStaticLayoutSpec.h"
+#import "ASStackLayoutSpec.h"
+
+NSString * const ASLayoutableValidationErrorDomain = @"ASLayoutableValidationErrorDomain";
+
+#pragma mark - Helpers
+
+static NSString *ASLayoutValidationWrappingErrorMessage(SEL selector, id obj, Class cl)
```

Comment:
There's some interchanging between the terms `ASLayoutValidation` and `ASLayoutableValidation`. I think `ASLayoutValidation` is the better choice.